fetch.js

```diff
@@ -283,6 +326,59 @@ const saveFetches = function (timeStarted, timeEnded, itemsInserted, err, result
 };
 
 /**
+ * Create a key using the current datetime
+ * @param {Date} d javascript Date() object
+ * @return {string} string in format yyyy-mm-dd/unixtime
+ */
+const getDateKey = function (d) {
```

Comment:
We've already got a dependency on `moment` so feel free to just use `const now = moment();` if you want.